actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -111,9 +111,9 @@ module ClassMethods
       #     protect_from_forgery except: :index
       #   end
       #
-      # You can disable forgery protection on controller by skipping the verification before_action:
+      # You can disable forgery protection on a controller using:
```

Comment:
```suggestion
      # You can disable forgery protection on a controller using skip_forgery_protection:
```

Since it is in the same module, "skip_forgery_protection" (without any markup) will be auto-linked to that method's documentation.